python/pyarrow/tests/parquet/test_parquet_file.py

```diff
@@ -353,3 +353,20 @@ def test_read_statistics():
                                 "null_count=1, distinct_count=None, "
                                 "min=-1, is_min_exact=True, "
                                 "max=3, is_max_exact=True>")
+
+
+def test_read_undefined_logical_type(parquet_test_datadir):
```

Comment:
So we're having the same test twice? (once here, once in `test_data_types.py`) Why is that?

Reply:
Ah, ok, fair enough.